internal/gengapic/genrest.go

```diff
@@ -91,13 +91,16 @@ func (g *generator) restClientInit(serv *descriptorpb.ServiceDescriptorProto, se
 	p("")
 	p("  // Points back to the CallOptions field of the containing %sClient", servName)
 	p("  CallOptions **%sCallOptions", servName)
+	p("")
+	p("  logger *slog.Logger")
```

Comment:
Could this and the import below be moved to `clientInit`?

Reply:
no, this is needed in by the concrete impl clients. It could also be added to that top level client, but I don't think there is a benefit.